### ISSUE TITLE
Braille input new bar after in-accord sign does not reset to voice 1

### ISSUE THREAD

**Issue by jrbowden** [musescore/MuseScore#20978]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Bug description

When inputting music using the Braille input feature, if you write music using multiple voices (called "in-accord" in braille terminology), the norm is to add voice 1, then the in-accord sign, then voice 2. This should repeat at each new bar.
Currently, when a new bar occurs, the voice remains voice 2. The in-accord sign in the second bar means that music is entered using voices 2 and 3 instead of 1 and 2. etc.
There seems no way to reset to voice 1 for each new bar, except to come out of the braille input window and select voice 1 manually.

### Steps to reproduce

1. Create a new score. We'll assume treble clef template. Key C major, time 4/4.
2. Ensure the Braille input panel is active (in Preferences)
3. Navigate to the braille window and enter the following braille music characters:

- Fifth octave C crotchet, C crotchet, 
- Fourth octave G crotchet, G crotchet,
- in-accord
- Fourth octave E minim, F minim

that is: 
⠨⠹⠹⠐⠳⠳⠣⠜⠐⠏⠟
 
or in key presses:
- J+L, F+J+K+L, F+J+K+L,
- K, F+D+K+L, F+D+K+L,
- F+D+L, S+J+K,
- K, F+D+S+J, F+D+S+J+K,

This bar works well and the music appears correctly in the print.

4. For the next bar, enter almost exactly the same as above, but change the second C crotchet (⠹ - F+J+K+L), to B crotchet (⠺ - D+J+K+L).
Notice this time that the notes are being entered for voices 2 and 3 instead of voices 1 and 2.

(Note: technically, in the input of the second bar, the fourth octave sign before G crotchet is not needed, but it won't harm).

### MuseScore Version

4.2.0

### Regression

No.

### Operating system

 Windows 10

### Additional context

When writing music in MuseScore's main score view, continuing to enter notes in several bars correctly maintains the same voice. It is arguable whether change voice commands should be used instead of the in-accord sign in the braille.
